$NetBSD: patch-aa,v 1.7 2013/12/10 08:27:06 obache Exp $

* Use libtool in the wrapper directory instead of the one installed by
  apr.  Otherwise, dependency_libs in libaprutil-1.la is set incorrectly.

  Don't let the build's LDFLAGS be polluted by what provides apr, because apr
  doesn't know about all dependencies of apr-util.  Otherwise the final binary
  wouldn't have the proper RPATHs.

--- configure.orig	2013-11-13 16:41:32.000000000 +0000
+++ configure
@@ -24308,7 +24308,7 @@ case $host in
             < $APR_BUILD_DIR/apr_rules.mk > $abs_builddir/build/rules.mk
         ;;
     *)
-        cp $APR_BUILD_DIR/apr_rules.mk $abs_builddir/build/rules.mk
+        sed -e 's:^LIBTOOL=.*:LIBTOOL=$(WRAPPER_DIR)/bin/libtool:' -e "s:^LDFLAGS=.*:LDFLAGS=${LDFLAGS}:" $APR_BUILD_DIR/apr_rules.mk > $abs_builddir/build/rules.mk
         ;;
 esac
 
